### PR TITLE
Updated to receive alerts for CC test channel

### PR DESCRIPTION
**Pull request by himanip94**
Added a route and receiver for CC KVM test channel alerts (#alert-kvm-test)

### CODE REVIEW

**Review by viennaa — CHANGES_REQUESTED** [review]
Bump the chart version please.

**Comment by himanip94** [comment]
Dumb question but could you please tell me which file are we talking about?

File: global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml [hunk 1]
```diff
@@ -467,6 +467,26 @@ spec:
...
+      # Test Channel for CC KVM Alerting slack_alert_kvm_test
+      - receiver: slack_alert_kvm_test
+        continue: true
+        matchers:
+          - name: tier
+            matchType: "="
+            value: vmware
+          - name: support_group
+            matchType: "="
+            value: compute
```
Comment (richardtief) [inline global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml:479]: These labels are not used at all. They can be removed. 
```suggestion
```

File: global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml [hunk 2]
```diff
@@ -467,6 +467,26 @@ spec:
...
+          - name: service
+            matchType: "="
+            value: kvm
+          - name: severity
+            matchType: "="
```
Comment (richardtief) [inline global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml:484]: Regex matches require a different `matchType`.
```suggestion
            matchType: "=~"
```

**Review by richardtief — CHANGES_REQUESTED** [review]
Looks good. A few little things before we can continue.

**Comment by richardtief** [comment]
Yes, exactly.
> Dumb question but could you please tell me which file are we talking about? https://github.com/sapcc/helm-charts/blob/master/global/prometheus-alertmanager-operated/Chart.yaml#L5 - Bumping this to 4.5.2?

**Pull request merged**